Modules/Common.lua

```diff
@@ -374,6 +377,39 @@ do
 	end
 end
 
+function mergeDB(srcDB, modDB)
```

Comment:
In general, `Common.lua` should be for functions that can be used on any table or number or other type.  Many of these functions you added are very specific to the `GlobalCache`.  A few options I'd rather see

1. Make the functions more generic (e.g. for `mergeDB` here, have it be `mergeTables` and call it twice for conditions and multipliers)
2. Move the functions to a separate file specifically for managing the GlobalCache (might be worth turning GlobalCache into a proper OO class potentially)
3. Some of the functions are only used in one place.  It might be better off having those functions in the file that uses them either as a local function or a function defined in that file instead of polluting the global namespace

Reply:
Added this feedback to a new issue to be addressed in a new PR